codegen/src/lib.rs

```diff
@@ -315,13 +315,23 @@ impl Sig {
                                                 parameters.push(Parameter::Component(query.len()));
                                                 let elem = &ty.elem;
                                                 if mutable {
+                                                    #[cfg(not(feature = "reexport"))]
                                                     query.push(
                                                         parse_quote!(::legion::TryWrite<#elem>),
                                                     );
+                                                    #[cfg(feature = "reexport")]
+                                                    query.push(
+                                                        parse_quote!(self::legion::TryWrite<#elem>),
+                                                    );
                                                 } else {
```

Comment:
Why not put the legion path in a variable? That way you only need one pair of `#[cfg(feature = ..)]`, and you avoid a lot of code duplication.

Reply:
because I didn't think about using a path alias 👍